applications/nrf_desktop/src/modules/Kconfig.dfu_mcumgr

```
@@ -43,6 +39,33 @@ config DESKTOP_DFU_MCUMGR_MCUBOOT_DIRECT_XIP
 	  boots the image with a higher version. The module does not confirm the
 	  newly updated image after a successful boot.
 
+choice DESKTOP_DFU_MCUMGR_BACKEND
+	prompt "Choose DFU backend"
+	default DESKTOP_DFU_BACKEND_MCUBOOT if BOOTLOADER_MCUBOOT
+	default DESKTOP_DFU_BACKEND_SUIT if SUIT
+
+config DESKTOP_DFU_BACKEND_MCUBOOT
+	bool "Use MCUboot as DFU backend"
+	select MCUMGR_GRP_IMG
+	select MCUBOOT_BOOTUTIL_LIB
+	select IMG_MANAGER
+	depends on BOOTLOADER_MCUBOOT
+	help
+	  This option enables the MCUboot bootloader as the backend for the MCUmgr DFU module.
+
+config DESKTOP_DFU_BACKEND_SUIT
+	bool "Use SUIT as DFU backend"
+	select MCUMGR_SMP_LEGACY_RC_BEHAVIOUR
+	select MGMT_SUITFU
+	select ZCBOR_CANONICAL
+	select SSF_SUIT_SERVICE_ENABLED
+	select MGMT_SUITFU_GRP_SUIT
```

To keep the related options next to each other and keep the alphabetical order

```suggestion
	select MGMT_SUITFU
	select MGMT_SUITFU_GRP_SUIT
	select SSF_SUIT_SERVICE_ENABLED
	select ZCBOR_CANONICAL	
```